app-editors/nvi/nvi-1.81.7_pre20161022.ebuild

```diff
@@ -0,0 +1,87 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools db-use flag-o-matic
+
+DESCRIPTION="Vi clone"
+HOMEPAGE="https://sites.google.com/a/bostic.com/keithbostic/vi"
+SRC_URI="https://repo.or.cz/nvi.git/snapshot/f462fedd26f78eec1998da4b9cba360095a6aa53.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/nvi-f462fed"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~mips ~ppc ~ppc64 ~sparc ~x86 ~amd64-fbsd ~x64-macos"
+IUSE="perl tcl unicode"
+
+DEPEND=">=sys-libs/db-4.2.52_p5:=
+	>=sys-libs/ncurses-5.6-r2:=
+	perl? ( dev-lang/perl )
+	tcl? ( >=dev-lang/tcl-8.5:0= )"
+
+BDEPEND="virtual/pkgconfig"
+
+RDEPEND="${CDEPEND}
```

Comment:
seen this just now - `CDEPEND` does not exist in this ebuild. Presumably you want to inherit `DEPEND` there?